annotation-fulltext/src/main/java/eu/europeana/annotation/fulltext/subtitles/SubtitleHandler.java

```diff
@@ -1,52 +1,64 @@
 package eu.europeana.annotation.fulltext.subtitles;
 
+import java.io.BufferedReader;
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
 
 import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Component;
 
 import com.dotsub.converter.exception.FileFormatException;
 import com.dotsub.converter.importer.SubtitleImportHandler;
-import com.dotsub.converter.importer.impl.DfxpImportHandler;
-import com.dotsub.converter.importer.impl.QtTextImportHandler;
-import com.dotsub.converter.importer.impl.WebVttImportHandler;
 import com.dotsub.converter.model.Configuration;
 import com.dotsub.converter.model.SubtitleItem;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
 
+import eu.europeana.annotation.config.SubtitleFormat;
+import eu.europeana.annotation.config.SubtitleFormats;
+
+@Component("subtitleHandler")
```

Comment:
please use xml configurations for definition of this bean in order to use the application configuration in a consistent way.  The component annotation is not needed

Reply:
Makes sense, I have left it in case of migration to be already there, but we can leave it for now.

Reply:
this has no effect now, if it is left for migration purposes it should be commented out